src/ts/components/ads/AdMessageLabel.ts

```diff
@@ -23,24 +44,29 @@ export class AdMessageLabel extends Label<LabelConfig> {
   configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    const clearText = () => {
-      this.setText('');
+    let config = this.getConfig();
+    let text = config.text;
+
+    let updateMessageHandler = () => {
+      this.setText(StringUtils.replaceAdMessagePlaceholders(i18n.performLocalization(text), null, player));
+    };
+
+    let adStartHandler = (event: AdEvent) => {
+      let uiConfig = (event.ad as LinearAd).uiConfig;
+      text = (uiConfig && uiConfig.message) || config.text;
```

Comment:
nit: IMO optional chaining is more readable
```suggestion
      text = uiConfig?.message || config.text;
```

Reply:
[d1b797f](https://github.com/bitmovin/bitmovin-player-ui/pull/787/commits/d1b797f727307d2e57f9253ee9efeca0f054acf0)